examples/benchmark/main.rs

```diff
@@ -448,19 +445,19 @@ fn run_test(
     }
 
     // Create a DirectStorage queue which will be used to load data into a buffer on the GPU.
-    let queue_desc = DSTORAGE_QUEUE_DESC {
-        SourceType: DSTORAGE_REQUEST_SOURCE_TYPE::DSTORAGE_REQUEST_SOURCE_FILE,
-        Capacity: DSTORAGE_MAX_QUEUE_CAPACITY,
-        Priority: DSTORAGE_PRIORITY::DSTORAGE_PRIORITY_NORMAL,
-        Name: null(),
-        Device: device.as_raw() as *mut _,
+    let mut queue_desc = DSTORAGE_QUEUE_DESC {
+        SourceType: DSTORAGE_REQUEST_SOURCE_FILE,
+        Capacity: DSTORAGE_MAX_QUEUE_CAPACITY as u16,
+        Priority: DSTORAGE_PRIORITY_NORMAL,
+        Name: PCSTR::null(),
+        // TODO: There was a way ("convention") to copy a non-ref-counted Interface into these ManuallyDrop APIs
```

Comment:
Can you research this convention before merging?

Reply:
Wasn't intending to get this merged before researching. Some digging later, it was indeed a transmute as written in another TODO.

https://github.com/microsoft/windows-rs/pull/2361#discussion_r1150799401